pages/glossary/highest-possible-contrast.md

```diff
@@ -5,8 +5,6 @@ unambiguous: true
 objective: true
 ---
 
-The highest value of the [contrast ratios](https://www.w3.org/TR/WCAG21/#dfn-contrast-ratio) between two sets of colors (A and B).
-To calculate the highest value, find the colors in each set with the largest and smallest 
-[relative luminance](https://www.w3.org/TR/WCAG21/#dfn-relative-luminance) (`A-lum-max`, `A-lum-min`, `B-lum-max`, `B-lum-min`). The
-highest possible contrast is then the larger of the [contrast ratios](https://www.w3.org/TR/WCAG21/#dfn-contrast-ratio)
-of `A-lum-max` with `B-lum-min` versus `A-lum-min` with `B-lum-max`.
+The highest value of the [contrast ratios][contrast ratio] between two sets of colors (A and B). That is, the highest [contrast ratio][] between one color of A and one color of B.
```

Comment:
I was under the impression that the previous changes to this definition were made to clarify the computation of the highest contrast ratio. This is now reverting those changes. Are you sure you want to do this?

Reply:
🤔 I felt that the new definition was too much saying the "how" rather than the end result, which goes against the goal of ACT rules (e.g. would a tool that does compute all pairwise contrasts have a correct implementation of a definition (and rule) that says to compare the max/min luminance?)

https://github.com/act-rules/act-rules.github.io/pull/1961#discussion_r1026510096

Reply:
I agree that aligns better with the principles of ACT rules.